pkg/apis/scorecard/v1alpha1/register.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package v1alpha1 contains API Schema definitions for the cache v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=scorecard
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
+)
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "scorecard", Version: "v1alpha1"}
```

Comment:
Should the `scorecard` group include a domain to make it more unique? I don't know if we have a convention for SDK APIs (if not, we probably should). As a point of reference, OLM currently uses `operators.coreos.com`.

Reply:
Yeah, it would be a good idea to include a domain. I'm not quite sure what to use though. Since we are creating this now, I feel like it would be better to go with an `openshift` URL (like the metering-operator, which uses `metering.openshift.io`) instead of a `coreos` URL.

Reply:
Oh yeah, I agree with using `openshift.io`. I'd be good with any of these:
* `operator-sdk.openshift.io`
* `operatorsdk.openshift.io`
* `osdk.openshift.io`

And the kinds are `ScorecardOutput` and `ScorecardOutputList`, so since "Scorecard" is in the kind name, I figure we don't need "scorecard" in the group?

Reply:
I'll use `osdk.openshift.io` for the domain